_pages/setup-wallets/paper.md

```diff
@@ -24,60 +27,71 @@ There is no private information in this PDF, having it only reveals that you're
 As you pull each word out, write it down on a piece of paper.
 In the end you will have 23 words that look like this (**do NOT use this seed phrase**):
+
 ```
 define rifle cliff summer priority ability chimney cotton tennis crash husband try trophy ring cook portion drift stool thank ceiling notice lesson foam
 ```
 
 ## Calculate the 24th Word and Other Seed Data using SeedPicker
+
 The 24th word cannot be calculated on paper or in your head, so you will need a secure computer
-for this step. This guide recommends the open source tool `SeedPicker`, which was designed for this purpose.    
+for this step. This guide recommends the open source tool `SeedPicker`, which was designed for this purpose.
 
 #### Practice First with a Dry Run
+
 Let's start with a dry run. As this is just for testing, you don't have to worry about security at all.
-Use the following *insecure* seed phrase (the word `zoo` repeated 23 times):
+Use the following _insecure_ seed phrase (the word `zoo` repeated 23 times):
+
 ```
-zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo 
+zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo
 ```
+
 (an unbiased monkey would not pull the same word out of a hat 23 times in a row)
 
 #### Calculate the 24th Word
+
 1. Visit [seedpicker.net](https://seedpicker.net)
 1. Enter the practice phrase (`zoo zoo zoo...`) and hit `Calculate!`
-1. You will see that the 24th word calculation result is `buddy`:  
-![](/assets/img/setup-paper-seedpicker-example-output.png){:width="70%" class="border_image"}  
+1. You will see that the 24th word calculation result is `buddy`:
+   ![](/assets/img/setup-paper-seedpicker-example-output.png){:width="70%" class="border_image"}
 
 #### Save the Output
-1. Write down the whole 24 word phrase (`zoo` repeated 23 times + `buddy` as the 24th word) *offline* on paper only.
-**Do not save this to any computer.**
-1. Scroll down to `Export Public Key Info To Specter Desktop` and click on the `Download` button. 
-This will download a `json` file to your computer, containing the extended public key info.
-Save the file to a DVD-R or USB drive to later share with Specter-Desktop.  
-![](/assets/img/setup-paper-seedpicker-save-json.png){:width="70%" class="border_image"}  
 
+1. Write down the whole 24 word phrase (`zoo` repeated 23 times + `buddy` as the 24th word) _offline_ on paper only.
+   **Do not save this to any computer.**
+1. Scroll down to `Export Public Key Info To Specter Desktop` and click on the `Download` button.
+   This will download a `json` file to your computer, containing the extended public key info.
+   Save the file to a DVD-R or USB drive to later share with Specter-Desktop.
+   ![](/assets/img/setup-paper-seedpicker-save-json.png){:width="70%" class="border_image"}
+
+> Note: You do not need to record the xpub, root key fingerprint, or derivation path on your paper wallet.
```

Comment:
Added this note. Is this correct? Might you need to write down the derivation path, to ensure that your new hardware wallet (if you make one using your seed phrase to create a third signing wallet) has access to the location of the funds?